test/fast/accountTree.test.ts

```diff
@@ -1,11 +1,18 @@
-import { TestAccountTreeFactory } from "../../types/ethers-contracts/TestAccountTreeFactory";
-import { TestAccountTree } from "../../types/ethers-contracts/TestAccountTree";
+import {
+    TestAccountTree,
+    TestAccountTree__factory
+} from "../../types/ethers-contracts";
 import { Tree, Hasher } from "../../ts/tree";
 import { ethers } from "hardhat";
 import { assert } from "chai";
 import { randHex, randomLeaves } from "../../ts/utils";
 import { ZERO_BYTES32 } from "../../ts/constants";
 
+type UpdateBatchLeafs = Parameters<TestAccountTree["updateBatch"]>[0];
+type CheckInclusionTreeWitness = Parameters<
+    TestAccountTree["checkInclusion"]
+>[2];
```

Comment:
This and the usage points below are the best way I could determine to allow compatibility with Solidity tuple types now being generated in TS land (uint156[2] => [BigNumberish, BigNumberish]).

Would be nice if we could verify in Node.js before making txn calls that the length of arrays is valid, but for now will have to rely on the txn failing.